src/AdminUtils.js

```diff
@@ -121,48 +165,47 @@ class AdminUtils {
   }
 
   remove(discord, message, params) {
+    if (params[0] == 'help') {
+      return message.reply('remove `<clip>`: \n' +
+          'Permanently deletes `<clip>` from the soundboard.');
+    }
     if (!this._paramCheck(message, params)){ return; }
-    if (!(Object.keys(files).indexOf(params[0]) > -1)) {
+    if (!fm.inLibrary(params[0])) {
       return log.debug(`File not found: ${params}`);
     }
 
     const clipName = params[0];
```

Comment:
Could move this above line 173, and then refer to `clipName` in the rest of the method instead of param[0].